### PR TITLE
adding -i and -d flags together on euroix datasource

### PR DESCRIPTION
**Pull request by afenioux**
adding -i and -d flags together on euroix datasource

### CODE REVIEW

**Comment by afenioux** [comment]
LGTM now, but should be better to squash and merge or rebase en merge

File: peerme/euroix_json.py [hunk 1]
```diff
@@ -178,9 +181,14 @@ def session_on_all_ixp(self):
                                                 if inetF == "ipv6" and optionals == "as_macro": my_peer.as_set_v6 = connection["vlan_list"][inetF][optionals]
                                 except KeyError:
                                     pass
-                            #special case if my_asn is declared (don't add it to the list)
-                            if (my_asn is None) or (my_asn != my_peer.asn ):
-                                peers_list.append( my_peer )
+                            #we don't want to add my_asn to the list
+                            if (int(my_asn) == my_peer.asn):
```
Comment (cooperlees) [inline peerme/euroix_json.py]: Int both to be sure?

**Review by cooperlees — APPROVED** [review]
Seems legit 😎

File: peerme/euroix_json.py (hunk 1 above)
Comment (afenioux) [reply peerme/euroix_json.py]: hum... let me check again

**Comment by afenioux** [comment]
It's now working fine

**Pull request merged**